nilearn/datasets/func.py

```diff
@@ -1152,4 +1152,74 @@ def fetch_abide_pcp(data_dir=None, n_subjects=None, pipeline='cpac',
         if ext == '.1D':
             files = [np.loadtxt(f) for f in files]
         results[derivative] = files
-    return Bunch(**results)
\ No newline at end of file
+    return Bunch(**results)
+
+
+def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
+                        verbose=0):
```

Comment:
The make_Xy should probably come back in this function, to make the examples easier. I would suggest naming it 'return_raw_data' and putting it to False by default.